tox.ini

```diff
@@ -52,6 +52,7 @@ passenv =
 extras =
     dev
     worker
+    fmv
```

Comment:
`kwiver` only has manylinux wheels which means we cannot run this natively on mac or windows. 

@ResonantGeoData/developers, if the tests break for you, this is likely why.

Reply:
And this will require ffmpeg